### ISSUE TITLE
Server Error on AR sites

### ISSUE THREAD

**Issue by ebju** [migcontrol/django-migcontrol#261]
I am trying to put text links into this AR text:
https://migration-control.info/ar/blog/%D8%A7%D8%B3%D9%85-%D8%A7%D9%84%D8%B5%D9%81%D8%AD%D8%A9-%D9%83%D9%85%D8%A7-%D8%B3%D8%AA%D8%B8%D9%87%D8%B1-%D9%81%D9%8A/

I did work sometimes, but now I regularly get a server-error notice
![Server error](https://github.com/migcontrol/django-migcontrol/assets/95855925/f90085ed-2e11-428c-b806-c62efc7a844c)

**Comment by benjaoming** [comment]
I don't understand Arabic, what does the message say? Maybe it's helpful...

**Comment by ebju** [comment]
Hey, hping you are well and sound!

I don't speak AR either.But I have tried the same thing from the EN
wagtail site. Same thing happens, and it sais something like "server out
of order".

Best greetings, eb

Am 15.02.2024 um 17:33 schrieb Benjamin Balder Bach:

**Comment by benjaoming** [comment]
Next time it happens, can you get somehow the exact error message that you see and send the time when it occurred? Will look it up then in the logs :+1:

**Comment by ebju** [comment]
![Server fehler](https://github.com/migcontrol/django-migcontrol/assets/95855925/c823d234-3a1b-43ed-b56f-5fe48ead16e8)

**Comment by benjaoming** [comment]
It seems to be an error related to the selected text. Either it's because the text is too long or because something crashes from a lack of support for the Arabic characters.

I would try - for now - to select a smaller text.

If that doesn't work, we can try to upgrade Wagtail. We're a few versions behind already. And the latest versions have improved support for right-to-left (Arabic!), especially for the text editor. This was announced quite recently.

If upgrading Wagtail isn't the answer, then we should submit the error "upstream" to the Wagtail project...

**Comment by benjaoming** [comment]
That's what Wagtail 6.0 promises :raised_hands: 

https://docs.wagtail.org/en/stable/releases/6.0.html#right-to-left-language-support

**Comment by ebju** [comment]
Yes this would be great. We also have a problem with AR texts, if the paragraph starts with an Latin Acronym...
Looking foreward to have these things solved, But, as I said, this is not urgent at all! Our (few) AR visitors will forgive us.

**Comment by benjaoming** [comment]
As of https://github.com/migcontrol/django-migcontrol/pull/269, we are now on Wagtail 5, so getting closer :)

**Comment by benjaoming** [comment]
Note when upgrading Wagtail: Check if it can delete several newsletter signups at once.

**Comment by benjaoming** [comment]
Does this work now? Reopen if it doesn't 👍
